ui/components/multichain/account-picker/account-picker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ButtonBase,
+  ICON_NAMES,
+  AvatarAccount,
+  Icon,
+  Text,
+} from '../../component-library';
+import {
+  Color,
+  FONT_WEIGHT,
+  Size,
+} from '../../../helpers/constants/design-system';
+
+export const AccountPicker = ({ account, onClick }) => {
+  return (
+    <ButtonBase
+      className="multichain-account-picker"
+      onClick={onClick}
+      backgroundColor={Color.transparent}
```

Comment:
Use `BackgroundColor` here I think this will help once this component is using TS
```suggestion
      backgroundColor={BackgroundColor.transparent}
```

Reply:
Done here: https://github.com/MetaMask/metamask-extension/pull/18177/commits/c2709281ba457b4dc7cb5816d56daf685df61a93